core/src/main/scala/kafka/log/LogConfig.scala

```diff
@@ -232,7 +232,7 @@ object LogConfig {
     }
   }
 
-  private val configDef: LogConfigDef = {
+  val configDef: LogConfigDef = {
```

Comment:
How about adding to `LogConfigDef`:

```scala
def serverConfigName(configName: String): Option[String] =
  serverDefaultConfigNames.get(configName)
```

And to the `LogConfig` object:

```scala
  def serverConfigName(configName: String): Option[String] =
    configDef.serverConfigName(configName)
```

Would that be enough for your use case?